tools/hyphy/hyphy_gard.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<tool id="hyphy_gard" name="HyPhy-GARD" version="@VERSION@+galaxy0">
+    <description>: Recombination Detection</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <version_command>HYPHYMP --version | tail -n 1 | awk '{print $1}'</version_command>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    export MPI_NODE_COUNT=\${GALAXY_SLOTS:-1} &&
+    ln -s '$input_file' gard_input.fa &&
+    echo `pwd`/gard_input.fa > tool_params &&
+    echo "010010" >> tool_params &&
+    echo $gencodeid >> tool_params &&
+    echo $rate_cond.rate >> tool_params &&
+    #if str($rate_cond.rate) != "1":
+        echo $rate_cond.rate_classes >> tool_params &&
+    #end if
+    echo `pwd`/"gard_input.fa.GARD.json" >> tool_params &&
+    @HYPHY_INVOCATION@ \$HYPHY_LIB/TemplateBatchFiles/GARD.bf > '$gard_log'
```

Comment:
We typically invoke GARD with `mpirun`; see [this line](https://github.com/veg/datamonkey-js-server/blob/master/app/gard/gard.sh#L40) of datamonkey-js-server. Do you think it will be required here? If so, should we add something like the [conda OpenMPI](https://anaconda.org/conda-forge/mpi) as a dependency for GARD?